habitat_baselines/rl/ppo/policy.py

```diff
@@ -24,7 +24,40 @@
 from habitat_baselines.utils.common import CategoricalNet, GaussianNet
 
 
-class Policy(nn.Module, metaclass=abc.ABCMeta):
+class Policy(abc.ABC):
+    action_distribution: nn.Module
+
+    def __init__(self):
+        pass
+
+    @property
+    def should_load_agent_state(self):
+        return True
+
+    @property
+    def num_recurrent_layers(self) -> int:
+        return 0
+
+    def forward(self, *x):
+        raise NotImplementedError
+
+    def act(
+        self,
+        observations,
+        rnn_hidden_states,
+        prev_actions,
+        masks,
+        deterministic=False,
+    ):
+        raise NotImplementedError
+
+    @classmethod
+    @abc.abstractmethod
+    def from_config(cls, config, observation_space, action_space):
+        pass
+
+
+class NetPolicy(nn.Module, Policy):
```

Comment:
Will the config keys be injected correctly into NetPolicy now what the name changed? I feel like the policy_config argument is missing from Policy.

Reply:
The policy config is passed to `NetPolicy`. What exactly do you think should be changed?

Reply:
I see, my bad. I thought what was going on was more complicated. If that is the case, I think the Policy class should be moved somewhere else. Since it is used in both `ppo` and `hrl`, it should live in the root folder `rl`